tasks/prelim.yml

```diff
@@ -243,11 +243,21 @@
       - RHEL-07-040420
       - ssh
 
-- name: "PRELIM | RHEL-07-020030 RHEL-07-020040 RHEL-07-021600 RHEL-07-021610 RHEL-07-021620 | Install AIDE"
-  yum:
-      name: aide
-      state: present
-  notify: init aide
+- name: "PRELIM | RHEL-07-020030 RHEL-07-020040 RHEL-07-021600 RHEL-07-021610 RHEL-07-021620 | Install and initialize AIDE"
+  block:
+      - name: "PRELIM | RHEL-07-020030 RHEL-07-020040 RHEL-07-021600 RHEL-07-021610 RHEL-07-021620 | Install AIDE"
+        yum:
+            name: aide
+            state: present
+        notify: init aide
+
+      - name: "PRELIM | RHEL-07-020030 RHEL-07-020040 RHEL-07-021600 RHEL-07-021610 RHEL-07-021620 | Check for existing AIDE database"
+        stat:
+          path: "{{ rhel7stig_aide_db_file }}"
+        register: rhel7stig_aide_db_status
+        check_mode: no
+        changed_when: not rhel7stig_aide_db_status.stat.exists
```

Comment:
Since we're conditionally notifying the handler here, we don't need the "when" on the handler itself.

Reply:
Good catch